V1_EngagingWithUs.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: engaging-with-us
+is: 'published'
+---
+
+## Engaging with us
+
+We are continually adding value and want to include you as early as possible in our development process so we can include 
+your input and feedback. Some features are stable (in that they will not change), others are in progress of being built which we call **beta** 
+and others are things we want to try and discover what would work best, we call **experimental**. Additionally, when we are about to 
+introduce something new and want to get feedback on the concepts and understanding before we expose API’s, so that your 
+feedback can inform and guide the future API, we will call these **future**. Also, **future** may, (but not always), include the ability for you to 
+play with an API, through one of our tools (like the GraphQL Explorer), so you can have some experience with it and give us 
+feedback to help inform and improve things.
+
+API’s that are going to be going out of service will be called **deprecated**, which means that while this is still available 
+for developers to use, it will be removed and calls to this API will no longer work. They will be kept on to provide backward 
+compatibility, and to provide time for developers to update affected code.
+
+You will see these various categories called out on those specific APIs. All other, undecorated API references are 
+considered published and are fully supported. 
+
+To provide more transparency in how we’re working and what we’re building, we will surface our content in markdown files in 
+the github repo - [github.com/watsonwork/watsonwork-developer-docs](https://github.com/watsonwork/watsonwork-developer-docs) so that you can engage with us through github to inform 
+and provide feedback to help make and design great APIs.
+
+### How we make our developer experience
+
+We build our developer experience [developer.watsonwork.ibm.com](https://developer.watsonwork.ibm.com) through the use of the files we 
+keep in the github repository mentioned above. When we build, we only bring in those files that are marked as **published**, **beta** 
+or **deprecated**, and do not add those marked as **future** or **experimental**.
+
+**Folder Structure**
+
+Root: 
+- [Navigation Menu](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/NavigationMenu.md)
+- [Get Started](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/V1_GetStarted.md)
+- [What's New](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/Whats_New.md)
+
```

Comment:
We have other entries in the root.... was the above a sample or is it supposed to be complete.

Reply:
I don't know what you mean here

Reply:
You are providing info on folder structure and call out 3 items at the "Root:" level (Navigation Menu, Get Started, What's New).  We have more than 3 items at the root level already.